src/ni/vsbuild/PipelineInformation.groovy

```diff
@@ -13,6 +13,10 @@ class PipelineInformation implements Serializable {
    }
 
    public void printInformation(script) {
-      script.echo "Pipeline will be run for LV versions $lvVersions and will execute on node(s) with label \'$nodeLabel\'."
+     String infoString = "Pipeline will be run for LV versions $lvVersions"
```

Comment:
The indentation is off for this entire method. Make sure each tab is 3 spaces. Also add an empty line after closing the if clause for style consistency.